src/components/Button/variations/Button.icon.tsx

```diff
@@ -18,7 +18,7 @@ const ButtonIcon: React.FC<ButtonIconProps> = button.attrs({
 	&& {
 		padding: ${tokens.space.xs};
 		min-height: unset;
-		border-radius: ${tokens.radii.circleRadius};
+		border: none;
```

Comment:
```suggestion
                 border: none;
		border-radius: ${tokens.radii.circleRadius};
```
Let's keep these round still, right?